src/index.d.ts

```diff
@@ -85,6 +85,16 @@ declare module 'lodash' {
       reject: () => void
     };
 
+    /**
+     * Part of `lodash-firecloud`.
+     *
+     * Checks if value is defined.
+     *
+     * @param value The value to check.
+     * @return Returns true if value is defined, else false.
+     */
+    isDefined(value: T | undefined): value is T;
```

Comment:
What does `value is T` mean? I would expect `Boolean` here.

Reply:
as per my (**limited**) understanding:

* T is a generic
* you return `true` (not just Boolean) if `value is T` and thus not `value is undefined`

original `isUndefined` for instance is defined similarly https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/lodash/common/lang.d.ts#L1504

similar pattern can be found here https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/lodash/fp.d.ts#L1908

Reply:
ok, I see now